packages/faceted-search/src/components/AddFacetPopover/AddFacetPopver.component.test.js

```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import { mount } from 'enzyme';
 import { act } from 'react-dom/test-utils';
-import { AddFacetPopover } from './AddFacetPopover.component';
+import { AddFacetPopover, filterByLabel } from './AddFacetPopover.component';
```

Comment:
In my opinion there is no need to test the filterByLabel unitary.
It's implicitly test in an higher test.

Reply:
Yes, i think you're right, this test is a little bit overkill.
I remove it